currency-converter/currency.html

```diff
@@ -116,6 +122,15 @@ <h2>Currency Rates</h2>
         </ul>
       </div>
     </div>
+    <footer>
+      <div class="footer">
+        <p>
+          &lt;&lt;&lt; &copy;
```

Comment:
Nowadays you can simply use `<`. Make sure to have `<meta charset="utf-8">` in the `<head>` element of your HTML file.